src/app/pages/FrontPage/FrontPage.jsx

```diff
@@ -110,7 +110,8 @@ const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
         <IndexPageContainer>
           {groups.map((group, index) => (
             <Fragment key={group.title}>
-              {group.type === 'useful-links' && renderMostRead()}
+              {group.type === 'useful-links' &&
+                renderMostRead(mostReadEndpointOverride)}
```

Comment:
Nice! Thanks for fixing this! I am assuming that it doesn't affect the rendering of the most read on the page (i.e. not via storybook?)